plugin/completion.py

```diff
@@ -260,7 +256,14 @@ def do_request(self, prefix: str, locations: 'List[int]') -> None:
         if not client:
             return
 
-        if settings.complete_all_chars or self.is_after_trigger_character(locations[0]):
+        prev_point = locations[0] - 1 if locations[0] - 1 >= 0 else 0
```

Comment:
Nitpick: you can use `prev_point = max(0, locations[0] - 1)` here.

Reply:
I dind't know that, I like it :D